docs/example.rst

```diff
@@ -54,8 +54,7 @@ and arguments are included for completeness.
     AUTH_LDAP_FIND_GROUP_PERMS = True
 
     # Cache group memberships for an hour to minimize LDAP traffic
```

Comment:
Please update this comment. This setting applies to more than just group membership. It also caches DNs.